model/clusters_mgmt/v1/control_plane_resource.model

```diff
@@ -14,13 +14,18 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Manages a specific upgrade policy.
+// Manages control plane resources.
 resource ControlPlane {
-	// Reference to the state of the upgrade policy.
+	// Reference to the collection of upgrade policies for the control plane.
 	locator UpgradePolicies {
 		target ControlPlaneUpgradePolicies
 	}
 
+	// Reference to the collection of log forwarders.
+	locator LogForwarding {
```

Comment:
@willkutler This should be `LogForwarders`. The resource is `LogFowarder`, so API convention is that the collection is `LogFowarders`.